vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -945,6 +944,21 @@ private void updateSelectedFragment(final TabLayout.Tab item) {
                 }
                 mCurrentFragmentTag = TAG_FRAGMENT_PEOPLE;
                 mSearchView.setQueryHint(getString(R.string.home_filter_placeholder_people));
+
+                mInviteContactLayout.setVisibility(View.VISIBLE);
+
+                mInviteContactLayout.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        // Create a new discussion
+                        // Invite one or more users
+                        // If only one contact is selected, it will be a direct chat
+                        // Multiselection mode is required
+                        // TODO sp3-11 invite only non Tchap users
+                        //DinsicUtils.alertSimpleMsg(VectorHomeActivity.this, getString(R.string.action_not_available_yet));
```

Comment:
Please remove this useless comment
// We launch a VectorRoomCreationActivity activity to invite some non-tchap contacts by using their email